users/tests/test_user_model.py

```diff
@@ -0,0 +1,110 @@
+import secrets
+import string
+
+import pytest
+from django.contrib.auth import get_user_model
+from django.test import Client
+from django.urls import reverse_lazy
+
+User = get_user_model()
+
+HTTP_SUCCESS = 200
+
+
+def generate_random_password(length: int = 12) -> str:
+    characters = string.ascii_letters + string.digits + string.punctuation
+    secure_random = secrets.SystemRandom()
+    return "".join(secure_random.choice(characters) for _ in range(length))
+
+
+@pytest.fixture()
+def generated_password() -> str:
+    return generate_random_password()
+
+
+@pytest.fixture()
```

Comment:
```suggestion
@pytest.fixture
```
This is not a function: https://docs.pytest.org/en/7.1.x/how-to/fixtures.html

Also please move to conftest.py as it will be global.